### DISCUSSION TITLE
Maintenance and Support

### DISCUSSION THREAD

**Discussion by interfrastic** [vsch/idea-multimarkdown#913] [General]
I purchased a subscription to Markdown Navigator last year and it is coming up for renewal next month.

I am thinking of cancelling because there does not seem to be any maintenance or support activity on this project; in fact, even the author’s website has gone offline:

<https://vladsch.com/product/markdown-navigator>

The last archived version of the site is from September 24, 2020:

<http://web.archive.org/web/20200924060428/https://vladsch.com/product/markdown-navigator>

So, is Markdown Navigator dead, with 3.0 the terminal version?

**Comment by vsch** [comment]
From the creeping integration of JetBrains bundled markdown plugin into their other plugins it sure looks like the days of Markdown Navigator are numbered.

For a more detailed explanation please see #918.
